app/assets/stylesheets/pin-index.css.scss

```diff
@@ -90,7 +92,7 @@ h2.category{
   margin-bottom: 10px;
   font-size: 10px;
   overflow-wrap: break-word;
-  color: $primary-background-color;
+  color: gray;
```

Comment:
The default color is too light, thus change it to gray to show
but will have to work with Dale to work out color schemes and then put in the primary color opacity function later.